src/util/csvParser/CsvParser.ts

```diff
@@ -0,0 +1,36 @@
+import { parse } from "csv-parse/sync";
+import { CsvErrorCode } from "./CsvErrors";
+
+export const parseCsv = (data, wantedColumns) => {
+  try {
+    const records = parse(data, {
+      columns: true,
+      skip_empty_lines: true
+    });
+    return records;
+  } catch (err) {
+    if (err.code === "CSV_RECORD_INCONSISTENT_COLUMNS") {
+      if (err.columns.length !== wantedColumns.length) {
+        throw {
+          code: CsvErrorCode.ColumnsLengthMissmatch,
+          expected: wantedColumns,
+          received: err.columns
+        };
+      }
+      for (const column of wantedColumns) {
+        if (!err.columns.some(element => element.name === column)) {
+          throw { code: CsvErrorCode.MissingColumn, column: column };
+        }
+      }
```

Comment:
this kind of error can be simplified in this manner: using this:

<img width="327" alt="image" src="https://user-images.githubusercontent.com/20487262/194730857-af43d395-0ee2-4122-8a7a-054e201c2ae4.png">

we can have a single kind of error like 

```
{
  code: "COLUMNS_LENGTH_MISSMATCH",
  missingColumns: ["a"],
  extraColumns: ["c"]
}
```